README.md

```diff
@@ -46,6 +59,10 @@ Keep in mind that now it is not possible to select the content with
 mouse, so apply the `cursor: default;` CSS style to prevent confusing
 the users (or even `cursor: grab;` in case the content is not a text).
 
+You can use `zoom-in(number)`,`zoom-out(number)` to zoom in and zoom out a `dragsimg`.  
+You have to put a number in `zoom-in(number)`,`zoom-out(number)`.  
+The recommended number is `20`.
```

Comment:
Either explain, what is the meaning of "20", or if you always suggest it to be 20, we don't need the argument at all, so that users could simply do `zoomIn()`.